src/main/java/io/github/wistefan/mapping/JavaObjectMapper.java

```diff
@@ -283,6 +265,23 @@ private <T> EntityVO buildEntity(T entity, Method entityIdMethod, Method entityT
 		return entityVO;
 	}
 
+	public <T> SubscriptionVO toSubscriptionVO(T subscription) {
+		isMappingEnabled(subscription.getClass())
+				.orElseThrow(() -> new UnsupportedOperationException(
+						String.format("Generic mapping to NGSI-LD subscriptions is not supported for object %s",
+								subscription)));
+
```

Comment:
You shoud never create an ObjectMapper in-line on call, its very slow. Better move the code to the EntityVOMapper, where the ObjectMapper is already available